src/libs/Localize/index.js

```diff
@@ -97,8 +88,10 @@ function arrayToString(anArray) {
     return aString;
 }
 
-export {
+const Localize = {
```

Comment:
I would remove this change to simplify the diff in this PR, and go back to using named exports for this module:

```js
export {
    translate,
    translateLocal,
    arrayToString,
};
```

And then you can import that like:

```js
// In an ES6 module
import * as Localize from 'src/libs/Localize';

// In a CJS module – no `.default` at the end
const Localize = require('src/libs/Localize');
```